src/theme.scss

```diff
@@ -15,15 +15,26 @@ body {
 $colors : (
   primary: #3359DF,
   secondary: #0A21A8,
+  primaryshade: #127dc5,
```

Comment:
I'm not sure the variable naming for the colors is fool-proof. Please consider refactoring, here is a useful resource http://chir.ag/projects/name-that-color/

Reply:
interesting read @andela-hms

Reply:
Thanks 👍  fixed